src/main/webapp/possibleInterviewTimes.jsp

```diff
@@ -0,0 +1,40 @@
+<%@ page import="java.util.List" %>
+<%@ page import="com.google.sps.data.PossibleInterviewSlot" %>
+<%
+  List<List<PossibleInterviewSlot>> list = (List<List<PossibleInterviewSlot>>) request.getAttribute("weekList");
+  pageContext.setAttribute("list", list);
+%>
+<%@ taglib uri="http://java.sun.com/jsp/jstl/core" prefix="c" %>
+
+<c:choose>
+  <c:when test= "${empty pageScope.list}">
+    <h2 style="text-align: center">No interviews are available at this time.</h2>
+  </c:when>
+  <c:otherwise>
+  <c:forEach items = "${pageScope.list}" var = "day">
```

Comment:
Shouldn't the for each loop be tabbed once?

Reply:
Done